Vagrantfile

```diff
@@ -0,0 +1,39 @@
+Vagrant.configure("2") do |config|
+  config.vm.box = "hashicorp/bionic64" 
+  config.vm.provision "shell", inline: <<-SHELL
+    sudo apt-get update
+
+    # Install pyenv prerequisites
+    sudo apt-get install -y build-essential libssl-dev zlib1g-dev libbz2-dev \
+    libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev \
+    xz-utils tk-dev libffi-dev liblzma-dev cifs-utils python-openssl git
+
+    # Install pyenv
+    git clone https://github.com/pyenv/pyenv.git /home/vagrant/.pyenv 
+    echo 'export PYENV_ROOT="/home/vagrant/.pyenv"' >> /home/vagrant/.profile
+    echo 'export PATH="$PYENV_ROOT/bin:$PATH"' >> /home/vagrant/.profile
+    echo 'eval "$(pyenv init -)"' >> /home/vagrant/.profile
+    export PYENV_ROOT="/home/vagrant/.pyenv"
+    export PATH="$PYENV_ROOT/bin:$PATH"
+    eval "$(pyenv init -)"
```

Comment:
Instead of repeating these three commands you can run the .profile file with `source /home/vagrant/.profile`